op-service/eth/ssz_test.go

```diff
@@ -250,21 +250,21 @@ func TestOPB04(t *testing.T) {
 
 func createPayloadWithWithdrawals(w *types.Withdrawals) *ExecutionPayload {
 	return &ExecutionPayload{
-		ParentHash:    common.HexToHash("0x123"),
-		FeeRecipient:  common.HexToAddress("0x456"),
-		StateRoot:     Bytes32(common.HexToHash("0x789")),
-		ReceiptsRoot:  Bytes32(common.HexToHash("0xabc")),
+		ParentHash:    common.HexToHash("0x1234"),
+		FeeRecipient:  common.HexToAddress("0x4567"),
+		StateRoot:     Bytes32(common.HexToHash("0x6789")),
+		ReceiptsRoot:  Bytes32(common.HexToHash("0xabcd")),
 		LogsBloom:     Bytes256{byte(13), byte(14), byte(15)},
-		PrevRandao:    Bytes32(common.HexToHash("0x111")),
+		PrevRandao:    Bytes32(common.HexToHash("0x1111")),
 		BlockNumber:   Uint64Quantity(222),
 		GasLimit:      Uint64Quantity(333),
 		GasUsed:       Uint64Quantity(444),
 		Timestamp:     Uint64Quantity(555),
-		ExtraData:     common.Hex2Bytes("0x666"),
+		ExtraData:     common.Hex2Bytes("0x6666"),
```

Comment:
This still winds up as an empty slice because `Hex2Bytes` doesn't support the `0x` prefix.